### PR TITLE
Data catalog url pop rb

### PR DESCRIPTION
**Pull request by dlangston**
You can delete the other PR (data-catalog-url-pop)

### CODE REVIEW

File: ooiui/templates/science/streams.html [hunk 1]
```diff
@@ -223,6 +222,27 @@
         $(".table").append( streamTableView.el );
     }
 
+function populate() {
+  // get url
+  var newURL = window.location.href;
+  console.log(newURL);
```
Comment (birdage) [inline ooiui/templates/science/streams.html]: @dlangston multiple console logs?

**Comment by DanielJMaher** [comment]
@dlangston I think Bird wants you to do something here

**Comment by birdage** [comment]
@dlangston clean up the console logs please.

**Comment by birdage** [comment]
@dlangston this needs some love

**Comment by DanielJMaher** [comment]
@dlangston You got stale hombre.  Need to remedy this situation!

**Pull request closed without merging**